mindsdb/integrations/handlers/rockset_integration/__init_.py

```diff
@@ -0,0 +1,23 @@
+from mindsdb.integrations.libs.const import HANDLER_TYPE
+
+from .__about__ import __version__ as version, __description__ as description
+try:
+    from .rockset_handler import (
+        RocksetIntegration as Handler,
+        connection_args_example,
+        connection_args
+    )
+    import_error = None
+except Exception as e:
+    Handler = None
+    import_error = e
+
+title = 'Rockset'
+name = 'rockset'
+type = HANDLER_TYPE.DATA
+icon_path = 'icon.svg'
```

Comment:
Please change this to correct icon name `icon.png`

Reply:
I must have done that and wrote over it.. Okay, fixing

Reply:
done